typed_ast/__init__.py

```diff
@@ -1 +1 @@
-
+__version__ = "1.1.0"
```

Comment:
Yeah, I know that's how mypy does it.  I figured that was too heavy-weight for typed_ast, which often goes just one or two commits between releases, but I guess it's not actually much additional work.  Do you think it's worth switching to that versioning scheme?